### PR TITLE
Fix/executiondata proto pkg

### PR DESCRIPTION
**Pull request by treethought**
Closes: #???

## Description

Fixes the package name for executiondata protobuf file from access -> executiondata.
Related to https://github.com/onflow/flow/pull/1275 and https://github.com/onflow/flow-go-sdk/issues/378 and needed for https://github.com/onflow/flow-go-sdk/pull/417

Previously, with package `access`, the `executiondata` types could not be imported in generated go code.

______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Comment by koko1123** [comment]
Hey this is a good idea because of the `access` package name overlap, but the intent behind the execution data APi was to be a subset of the Access API. The generated go files will break the `flow-go` implementation, so we have two possible routes around it:
1. Modify the corresponding handlers in `flow-go` to avoid copying mutex values
2. re-generate the go files with the older version of `buf` (in go.mod of this repo)

I suggest making a PR in `flow-go` for 1., then merging this PR, then updating the `flow-go` PR with the merged PR hash as a dependency

File: protobuf/README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Check out the [Flow Access API specification](/docs/content/access-api.md).
 You can use [buf](https://github.com/bufbuild/buf) to generate gRPC client stubs in a variety of languages.
 Please make sure you have `protoc-gen-go-grpc` installed, for example using command
 ```shell script
+go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
```
Comment (peterargue) [inline protobuf/README.md:14]: I'm not opposed to making this change, but I think we should do it as a separate PR

Comment (treethought) [reply protobuf/README.md]: I had added that line for what I used since I couldn't find what version was being used, but just found via https://github.com/onflow/flow/pull/905#issuecomment-1097221140 that it seems to be set in the flow-go makefile as [v1.3.2 ](https://github.com/onflow/flow-go/blob/master/Makefile#L84)

**Review by peterargue — APPROVED** [review]

File: protobuf/README.md (hunk 1 above)
Comment (peterargue) [reply protobuf/README.md]: ah ok. thanks for fixing the docs!

**Comment by peterargue** [comment]
FYI @nvdtf, @Guitarheroua
